### PR TITLE
[#1149] Implement Physical Address in MyProfile Page

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses issue https://github.com/episphere/connect/issues/1149

### CODE REVIEW

File: js/event.js [hunk 1]
```diff
@@ -263,64 +263,65 @@ export const addEventPOBox = () => {
     });
 };
 
-const onBlurPhysicalAddressLine = (event) => {
-    const physicalAddressCity = document.getElementById("physicalAddressCity");
-    const physicalAddressState = document.getElementById(
-        "physicalAddressState"
+const onBlurPhysicalAddressLine = (event, id) => {
+    console.log("onBlurPhysicalAddressLine")
```
Comment (Gbarra9) [inline js/event.js:267]: Console log might not be needed here.

**Review by Gbarra9 — APPROVED** [review]
Looks good. I like the updates to the variable names! Renaming 'physical' to 'UP' (short for user profile) makes the code clearer in the context of UP inputs.

File: js/settingsHelpers.js [hunk 2]
```diff
@@ -285,22 +287,30 @@ export const validateContactInformation = (mobilePhoneNumberComplete, homePhoneN
     hasError = true;
   }
 
-  if (!preferredEmail || !validEmailFormat.test(preferredEmail)) {
-    errorMessage('newPreferredEmail', translateText('settingsHelpers.emailFormat'), focus);
+  const emailValidation = await emailAddressValidation({
+      emails: {
+          upEmail: preferredEmail,
+          upEmail2: additionalEmail1 ? additionalEmail1 : undefined,
+          upAdditionalEmail2: additionalEmail2 ? additionalEmail2 : undefined,
```
Comment (we-ai) [inline js/settingsHelpers.js:294]: Better not use `undefined` as values.

**Review by we-ai — APPROVED** [review]
Looks good. See comments.

**Pull request merged**